Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewController.swift

```diff
@@ -638,7 +638,6 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
   }
 
   private func goToRewards(project: Project, refTag: RefTag?) {
-    /// Render rewards carousel that has the shipping location dropdown
```

Comment:
Removing this comment. It's no longer needed now that the no-shipping flow is the standard.